website/common/script/content/subscriptionBlocks.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
+
+let subscriptionBlocks = {
+  basic_earned: { // eslint-disable-line camelcase
```

Comment:
if you don't want to repeat the esline-disable-... you can write `/* eslint-disable camelcase */` at the top of the file once